src/pyiodaconv/def_jedi_utils.py

```diff
@@ -10,15 +10,18 @@
 import numpy as np
 import pyiodaconv.ioda_conv_engines as iconv
 import time
-from datetime import datetime
+from datetime import datetime, timezone
 
 metaDataName = iconv.MetaDataName()
 obsValName = iconv.OvalName()
 obsErrName = iconv.OerrName()
 qcName = iconv.OqcName()
 
+# Define metaData and ensure epoch object is in UTC
 iso8601_string = "seconds since 1970-01-01T00:00:00Z"
 epoch = datetime.fromisoformat(iso8601_string[14:-1])
+epoch = epoch.replace(tzinfo=timezone.utc)
```

Comment:
sigh,, it looks like ensuring UTC changed some other ctest output files:

```
test_iodaconv_icartt..................................................................failed
test_iodaconv_viirs_j1_l1b_albedo.....................................................failed
test_iodaconv_atms_nc4................................................................failed
test_iodaconv_atms_BGRemap_nc4........................................................failed
test_iodaconv_ghcn_snod...............................................................failed
```

we should likely separate `epoch` into it's own PR then.  But leave the import of `epoch` in the new converter does this sound acceptable @mer-a-o ?